yt/frontends/rockstar/data_structures.py

```diff
@@ -15,16 +18,49 @@
 
 
 class RockstarBinaryFile(HaloCatalogFile):
+    header: dict
+    _position_offset: int
+    _member_offset: int
+    _Npart: npt.NDArray[np.int64]
```

Comment:
Here's the portable way to express this type (where `Any` must be imported from `typing`)
```suggestion
    _Npart: "np.ndarray[Any, np.dypte[np.int64]]"
```